src/tracks/Pattern.cpp

```diff
@@ -1196,15 +1198,11 @@ void PatternView::paintEvent( QPaintEvent * )
 
 	p.setFont( pointSize<8>( p.font() ) );
 
-	QColor text_color = ( m_pat->isMuted() || m_pat->getTrack()->isMuted() )
-		? QColor( 30, 30, 30 )
-		: textColor();
-
 	if( m_pat->name() != m_pat->instrumentTrack()->name() )
 	{
-		p.setPen( QColor( 0, 0, 0 ) );
+		p.setPen( textShadowColor() );
 		p.drawText( 4, p.fontMetrics().height()+1, m_pat->name() );
-		p.setPen( text_color );
```

Comment:
Probably benign, but can you explain the before and after here in regards to the way muted text color was/is handled?